### PR TITLE
Profile page

### PR DESCRIPTION
**Pull request by SaraVieira**
sowy

### CODE REVIEW

**Comment by lbogdan** [comment]
What's with the `lodash-es` change to `lodash`?

File: packages/common/src/components/WideSandbox/elements.ts [hunk 1]
```diff
@@ -19,13 +19,24 @@ export const Container = styled.div`
   overflow: hidden;
   cursor: pointer;
 
-  margin-bottom: 2rem;
```
Comment (CompuIves) [inline packages/common/src/components/WideSandbox/elements.ts]: I believe this one needs to be back, or we need to change it in the explore page. Right now the cards in the explore page are glued to eachother.

File: packages/common/src/components/WideSandbox/elements.ts [hunk 2]
```diff
@@ -84,7 +95,7 @@ export const SandboxInfo = styled.div`
   left: -1px;
   right: -1px;
   padding: 0.75rem;
-  height: 130px;
```
Comment (CompuIves) [inline packages/common/src/components/WideSandbox/elements.ts]: There is some visual glitchiness if this one is removed in the explore page. Maybe we can put it back to `130px`? Not sure what breaks for the profile page in that case.

File: packages/dynamic-pages/pages/_document.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+          <meta property="twitter:image:height" content="630" />
+          <meta property="twitter:card" content="summary_large_image" />
+          <meta property="twitter:site" content="@codesandbox" />
+          <meta property="twitter:creator" content="@codesandbox" />
+          <meta name="viewport" content="width=device-width, initial-scale=1" />
+          <link
+            href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,700|Poppins:400,700|Source+Sans+Pro"
+            rel="stylesheet"
+          />
+          <link
```
Comment (CompuIves) [inline packages/dynamic-pages/pages/_document.js]: We can probably remove this one

File: packages/dynamic-pages/pages/_document.js [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+          <meta property="og:image:width" content="1200" />
+          <meta property="og:image:height" content="630" />
+          <meta property="twitter:image:width" content="1200" />
+          <meta property="twitter:image:height" content="630" />
+          <meta property="twitter:card" content="summary_large_image" />
+          <meta property="twitter:site" content="@codesandbox" />
+          <meta property="twitter:creator" content="@codesandbox" />
+          <meta name="viewport" content="width=device-width, initial-scale=1" />
+          <link
+            href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,700|Poppins:400,700|Source+Sans+Pro"
```
Comment (CompuIves) [inline packages/dynamic-pages/pages/_document.js]: Source Sans Pro is not needed here.

File: packages/dynamic-pages/.env [hunk 5]
```diff
@@ -0,0 +1 @@
+AIRTABLE=keyJugfwdJzOyL7Aa
```
Comment (CompuIves) [inline packages/dynamic-pages/.env]: Is this allowed to be public?

File: packages/common/src/components/Footer.tsx [hunk 6]
```diff
@@ -0,0 +1,176 @@
...
+  position: relative;
+  background-color: ${props => props.theme.background2.darken(0.2)};
+  padding: 1rem;
+  z-index: 5;
+`;
+
+const BasComponent = () => (
+  <a
+    id="bas"
+    href="https://www.aedin.com/in/basbuursma/"
```
Comment (CompuIves) [inline packages/common/src/components/Footer.tsx]: Did you replace `link` with `a`? 😂

File: packages/common/src/components/WideSandbox/elements.ts (hunk 1 above)
Comment (SaraVieira) [reply packages/common/src/components/WideSandbox/elements.ts]: this file doesn0t exist anymore :/

Comment (SaraVieira) [reply packages/common/src/components/WideSandbox/elements.ts]: Added back in common

File: packages/dynamic-pages/pages/_document.js (hunk 3 above)
Comment (SaraVieira) [reply packages/dynamic-pages/pages/_document.js]: done!

File: packages/dynamic-pages/pages/_document.js (hunk 4 above)
Comment (SaraVieira) [reply packages/dynamic-pages/pages/_document.js]: Removed

File: packages/dynamic-pages/.env (hunk 5 above)
Comment (SaraVieira) [reply packages/dynamic-pages/.env]: It's already in the repo ahah

File: packages/common/src/components/Footer.tsx (hunk 6 above)
Comment (SaraVieira) [reply packages/common/src/components/Footer.tsx]: Also don't know why bas has it's own component ahah

File: packages/common/src/components/WideSandbox/elements.ts (hunk 2 above)
Comment (SaraVieira) [reply packages/common/src/components/WideSandbox/elements.ts]: It deosn't break. It just has more empty space if there is no description

Added back

**Comment by SaraVieira** [comment]
All glitches and lack of spacing seem to be fixed in the latest deploy :D

**Comment by CompuIves** [comment]
Nice!

**Pull request merged**